org-roam.el

```diff
@@ -706,6 +700,15 @@ If there is no file with that ref, a file with that ref is created.
 
 This function is used solely in Org-roam's capture templates: see
 `org-roam-capture-templates'."
+  (org-capture-put :template
+                   (concat
+                    (org-roam--fill-template (or (org-capture-get :head)
+                                                 org-roam--capture-header-default)
+                                             org-roam--capture-info)
+                    "\n" (org-capture-get :template)))
+
+  (org-capture-put :immediate-finish org-roam--capture-immediate-finish)
```

Comment:
not sure if this is necessary though.

Reply:
`org-roam--capture-immediate-finish` is used as follows
1. It's value is set to the first value passed to `org-roam-capture`, `t` or `nil` otherwise
2. We want it to be `t` when we create a new file using `org-roam-find-file` to finish the capture immediately (this simulates the original behaviour of `org-roam`
3. I forgot to mention another org-capture-finilize-hook added, if this variable is set to `t`, we will switch to the newly created file's buffer, again to simulate the original `org-roam-find-file` behaviour
4. It's value is `nil` in all other cases

Reply:
This and other smaller adjustments are needed to simulate the original org-roam behaviour,
in particular, immediately switching to the buffer newly created by `org-roam-find-file` but staying in the same buffer when creating a file with `org-roam-insert`.

Reply:
I'd prefer changing the behaviour of the immediate finish in org-roam-capture for org-roam-find-file, because it's unintuitive. The org capture should go through, and the file should be visited at the end. That's the current behaviour in master now actually after the change earlier.

Reply:
Yes sure, fully agree. My patches are based on yesterday's develop branch, so I didn't have a chance to check up the new version. The things are changing blazingly fast :)